gms/inet_address.hh

```diff
@@ -58,9 +58,9 @@ public:
     bytes_view bytes() const noexcept {
         return bytes_view(reinterpret_cast<const int8_t*>(_addr.data()), _addr.size());
     }
-    friend inline bool operator==(const inet_address& x, const inet_address& y) noexcept = default;
-    friend inline bool operator<(const inet_address& x, const inet_address& y) noexcept {
-        return x.bytes() < y.bytes();
+    constexpr bool operator==(const inet_address&) const noexcept = default;
```

Comment:
Isn't this operator auto-generated from the <=> one?

Reply:
It is, but explicit operator== can be faster.